mlir/utils/jenkins/miopen-tests/miopen_validate.sh

```diff
@@ -78,8 +94,8 @@ function construct_fixed_args() {
        exit 2
     fi
     case "$DTYPE" in
-        fp16 ) FIXED_CONFIG_ARGS+=("convfp16") ;;
-        fp32 ) FIXED_CONFIG_ARGS+=("conv") ;;
+        fp16 ) FIXED_CONFIG_ARGS=("convfp16") ;;
```

Comment:
Why this change?

Reply:
To reset FIXED_CONFIG_ARGS every time construct_fixed_args() is called. For the tune-all case, construct_fixed_args() is called multiple times.

Reply:
Makes sense